app/Http/Requests/UserRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:'.config('users.name.max'),
```

Comment:
convention này 
```suggestion
            'name' => 'required|max:' . config('users.name.max'),
```
Sửa tương tự những hàm bên dưới

Reply:
đã fix